solidstart-tanstackquery-tailwind-modules/src/components/PRAndIssuesListItem/PRAndIssuesListItem.tsx

```diff
@@ -9,20 +9,23 @@ import {
   CommentIcon,
   ClosedIssueIcon,
 } from '../Icons';
-import { type } from '../PRAndIssuesHeader';
-import { Issue } from '~/types/issues-type';
+import { Issue, IssueNodeProps } from '~/types/issues-type';
+import { PullRequest, PullRequestNodeProps } from '~/types/pull-request-type';
+import { Label } from '~/types/label-type';
 
-const PRAndIssuesListItem = (props: Issue) => {
-  const [local] = splitProps(props, [
-    'number',
-    'title',
-    'url',
-    'state',
-    'createdAt',
-    'login',
-    'commentCount',
-    'labels',
-  ]);
+interface PRAndIssuesListItemProps {
+  issue?: Issue;
+  pullRequest?: PullRequest;
+}
+
+const PRAndIssuesListItem = (props: PRAndIssuesListItemProps) => {
+  const [local] = splitProps(props, ['issue', 'pullRequest']);
+
+  const item = local.issue || local.pullRequest;
+
+  console.log(item);
```

Comment:
There is some kind of problem with this file I think.

Before fixing it up locally, it would stop HRM from working and when I would make a change it would do some sort of timeout and Id have to shut down and restart the dev server after every change 😑

Reply:
I guess it's a problem with SolidStart itself, I also started a small CRUD project to learn SolidStart before this project and I had the same issue